homeassistant/components/asuswrt/router.py

```diff
@@ -341,7 +341,7 @@ async def close(self) -> None:
         """Close the connection."""
         if self._api is not None:
             if self._protocol == PROTOCOL_TELNET:
-                await self._api.connection.disconnect()
```

Comment:
I'll take it up with the aioasus package to actually just do this on destruction of the `TelnetConnection` such that in future this can be removed..

Reply:
So what is your suggestion, remove the call at all?

Reply:
For this hotfix, just calling it correctly is good to get things working again. You can do cleanup in a new PR that will be included in the normal release cycle.

Reply:
Think we can wait for new upstream version.